### DISCUSSION TITLE
Can I import JSON from Trello?

### DISCUSSION THREAD

**Discussion by KevinGIRAULT** [yashs662/rust_kanban#15] [Q&A]
I've exported a Kanban board from Trello to JSON and I'm looking for a tool in TUI that can import it.

Is rust_kanban able to do this? I haven't found anything to that effect, only that it can export in this format.

Thanks in advance for responses

**Comment by yashs662** [comment]
That feature although not implemented yet is on the roadmap. I want different services to seamlessly integrate with my project. Expect it in the coming weeks!!
